features/search_type.feature

```diff
@@ -0,0 +1,15 @@
+#language: fr
+
+Fonctionnalité: Chercher un type dans une glose
+
+Scénario: avec un type qui existe
+
+  Soit un document dont je suis l'auteur affiché comme glose
+  Quand je cherche le type "Ethnography/Interview"
```

Comment:
The scenario would be more interesting if by searching "ethno" you would find several types (see [here](https://github.com/Hypertopic/Cassandre/issues/86) for examples) but no types unrelated with ethnography (e.g. "translation"...).

Reply:
Does it work also with "Interview" (suffix rather than prefix)?

Reply:
We change the first scenario so that it search for "ethno" and we create a second test that search for  "Inter" to test with suffix as well